src/core/qgsvectorlayer.h

```diff
@@ -23,12 +23,15 @@
 #include <QSet>
 #include <QList>
 #include <QStringList>
+#include <Qcolor>
```

Comment:
Not even sure it needs to be there now.

On Fri, Jul 17, 2015 at 9:16 AM, Nyall Dawson <notifications@github.com>
wrote:

> In src/core/qgsvectorlayer.h
> <https://github.com/qgis/QGIS/pull/2209#discussion_r34848796>:
>
> > @@ -23,12 +23,15 @@
> >  #include <QSet>
> >  #include <QList>
> >  #include <QStringList>
> > +#include <Qcolor>
>
> I get a build error on this line... should be QColor
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/qgis/QGIS/pull/2209/files#r34848796>.
>